squad/frontend/static/squad/table.js

```diff
@@ -0,0 +1,3 @@
+$('#tests-table').floatThead({
+    position: 'auto'
+});
```

Comment:
I would rather apply this to all tables, so that the .js could be loaded only once in the base template instead of every page that has a table.

Reply:
the table header needs to be style with CSS to have a white background: for me here it has a transparent backrgound which doesn't look nie.

Reply:
I liked transparent :)

Reply:
fixed